docs/project_pilot_data_synchronization/about.md

```diff
@@ -0,0 +1,32 @@
+---
+id: about
+title: Pilot Data Synchronization
+custom_edit_url: null
+---
+
+
+## Overview
+
+The Flight Simulator Data Synchronization Project is a research initiative aimed at developing a real-time data synchronization channel between flight simulators and the iMotions human behavior research platform. This project will be conducted in two phases: a planning and design phase, followed by a testing phase. The goal is to enable the seamless transmission of critical flight data such as altitude, airspeed, and control surface deflections from Flight Simulation Training Devices (FSTDs) to iMotions, facilitating the real-time analysis of pilot workload, attention, and task management.
+
+This interdisciplinary project involves collaboration between Aeronautics students, who will act as system analysts, and Computer Science students, who will serve as software developers. The students will gain hands-on experience in both software design and validation. The project addresses a significant challenge identified in previous research at the Department of Aviation Science, where post-hoc synchronization of flight simulator data with biometric signals was time-consuming and inefficient. By establishing a Transmission Control Protocol (TCP) connection and leveraging advanced software tools, this project seeks to create a stand-alone multimodal physiological test battery for primary flight training. This will enable the aviation training community to better understand and improve pilot training effectiveness by visualizing and measuring the synergy between task management, attention, and workload in real-time.
+
+### Information
+
+- **Client** Gajapriya Tamilselvan - Aviation Science
+- **Current Tech Lead:** Harshitha Thota [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/harshitha-thota)
+- **Developers:**
+ TBD
+  
+- **Start Date:** August 2024
+
+- **Technologies Used:**
+  - C++
+
+- **Type:** TBD
+- **License:** [MIT](https://opensource.org/license/mit)
```

Comment:
Confirm that with Dr. Tamilselvan. It has implications for her. Also, we typically use LGPL as a default license. If MIT is preferred, I want to document why it's being chosen instead.

Reply:
I have made the necessary changes based on your feedback. Could you please review the updated code again. Thank you for your suggestions.